File: examples/calculator.js
/** @format */

const calculate = (A, B, operator) => {
  const firstNumber = A;
  const SecondNumber = B;

  switch (operator) {
    case "+":
      result = firstNumber + SecondNumber;
      break;
    case "-":
      result = firstNumber - SecondNumber;
      break;
    case "*":
      result = firstNumber * SecondNumber;
      break;
    case "/":
      result = firstNumber / SecondNumber;
      break;
    default:
      throw new Error("Invalid operator");
  }

  return result;
};

console.log(calculate(10, 4, "+"));

File: 00_Basic_Challanges/00_examples/calculate/calculator.py
## Basic calculater that takes two numbers and do basic oparation

def calculate(a, b, operator):
    """Calculate simple math operation on 2 numbers"""
    first_number = a 
    second_number = b
    
    if operator == "+":
        result = first_number + second_number
    elif operator == "-":
        result = first_number - second_number
    elif operator == "*":
        result = first_number * second_number
    elif operator == "/":
        result = first_number / second_number
    else:
        raise ValueError("Invalid operator")
        
    return result

print(calculate(10, 4, "+"))